vignettes/convert.qmd

```diff
@@ -15,13 +15,70 @@ execute:
   eval: false
 ---
 
-This vignette shows how to download OD datasets (v1 and v2) and convert them into analysis-ready formats.
-It does not cover the contents of the datasets.
-For that, see the codebooks in the [Codebook and cookbook for v1 (2020-2021) Spanish mobility data](v1-2020-2021-mitma-data-codebook.qmd) and in [Codebook and cookbook for v2 (2022 onwards) Spanish mobility data](v2-2022-onwards-mitma-data-codebook.qmd) vignettes.
+# Introduction {#intro}
 
-There are two main ways to import the datasets: as in-memory object with `spod_get()` or as a connection to DuckDB or Parquet files on disc with `spod_convert()`. The latter is recommended for large datasets, as it is much faster and more memory efficient.
+**TL;DR: For analysing more than 1 week of data, use `spod_convert()` to convert the data into `DuckDB` and `spod_connect()` to conenct to it for analysis using `{dplyr}`. Skip to the [section about it](#duckdb).**
 
-Load the package as follows:
+Using origin-destination data as an example, this vignette shows how to download the mobility datasets from versions v1 (2020-2021) and v2 (2022-onwards) of the Spanish mobility data and convert them into analysis-ready formats. See description of datasets in the [Codebook and cookbook for v1 (2020-2021) Spanish mobility data](v1-2020-2021-mitma-data-codebook.qmd) and in the [Codebook and cookbook for v2 (2022 onwards) Spanish mobility data](v2-2022-onwards-mitma-data-codebook.qmd).
```

Comment:
@eugenividal Actually yes, this "as an example" is here exactly because I changed the title to "Mobility" instead of "OD". So as for me, what is missing here is perhaps a sentence mentioning that working with other types of data available through the functions is similar.

Reply:
Totally agree!